unit_testing/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
We could consider adding `"private": true,` to the package.json in order to prevent accidental publishing but it's fine to leave it out.